helpdesk_mgmt_maintenance/models/maintenance_equipment.py

```diff
@@ -0,0 +1,39 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models, fields, api
+
+class MaintenanceEquipment(models.Model):
+    _inherit = "maintenance.equipment"
+
+    ticket_ids = fields.Many2many('helpdesk.ticket', string = "Tickets")
+
+    allow_ticket = fields.Boolean(string="Allow Tickets", default=False)
+
+    num_tickets = fields.Integer(string="Tickets", compute="_compute_num_tickets", store=True)
+
+
+    # NUMBER TICKETS
+
+    @api.depends("ticket_ids")
+    def _compute_num_tickets(self):
+        for equipment in self:
+            equipment.num_tickets = len(equipment.ticket_ids)
+
+    def action_view_tickets_equipment(self):
+        action = {
+            'name':('Tickets'),
+            'res_model': 'helpdesk.ticket', 
+            'view_mode': 'tree,form',
+            'type': 'ir.actions.act_window',
+            'domain':  [('id', 'in', self.ticket_ids.ids)],
```

Comment:
For default equipment filling when creating a new ticket, try this (not tested):
```suggestion
            'domain':  [('id', 'in', self.ticket_ids.ids)],
            'context': {'default_equipment_ids': [active_id]},
```